Trumpf.Coparoo.Web/Controls/Select.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace Trumpf.Coparoo.Web.Controls
+{
+    using System;
+
+    using OpenQA.Selenium;
+    using OpenQA.Selenium.Support.UI;
+
+    /// <summary>
+    /// Select control object.
+    /// </summary>
+    public class Select : ControlObject
```

Comment:
In the `Select` class I currently miss a property `public IEnumerable<Option> Options { get { ... } }`, which would, however, require an additional `Option` control object. Using these two classes one could select an item, e.g., like this: `MySelection.Options.First(selector).Select()` where `selector` is
- `option => option.Text == "foo"`,
- `option => option.Index == 4`, or
- `option => option.Value == "ara"`.

The `public SelectElement SelectElement` property and the three `SelectBy...` methods could then be removed. Supposed the `Option` exposes a `bool IsSelected {get;}` property, the `string Text` property could also be removed, as the same value could be retrieved, e.g., like this: `MySelection.Options.First(option => option.IsSelected).Text`. In the same manner, we could remove the `Index` and `Value` properties.